id: PYSEC-0000-CVE-2025-26699
details: An issue was discovered in Django 5.1 before 5.1.7, 5.0 before 5.0.13, and
  4.2 before 4.2.20. The django.utils.text.wrap() method and wordwrap template filter
  are subject to a potential denial-of-service attack when used with very long strings.
affected:
- package:
    name: django
    ecosystem: PyPI
    purl: pkg:pypi/django
  ranges:
  - type: ECOSYSTEM
    events:
    - introduced: "5.1"
    - fixed: 5.1.7
    - introduced: "5.0"
    - fixed: 5.0.13
    - introduced: "4.2"
    - fixed: 4.2.20
references:
- type: ARTICLE
  url: https://www.djangoproject.com/weblog/2025/mar/06/security-releases/
- type: WEB
  url: https://docs.djangoproject.com/en/dev/releases/security/
- type: WEB
  url: https://groups.google.com/g/django-announce
- type: WEB
  url: http://www.openwall.com/lists/oss-security/2025/03/06/12
- type: WEB
  url: https://lists.debian.org/debian-lts-announce/2025/03/msg00012.html
aliases:
- CVE-2025-26699
modified: "2025-03-19T20:15:19Z"
published: "2025-03-06T19:15:27Z"
